fix: argocd-tls-certs-cm is overwritten on any change (#553)
argoproj-labs/argocd-operator: docs/reference/argocd.md

    initialCerts: []
```

> ### IntialCerts Example

Reviewer: :+1:

> Initial Certs to configure Argo CD to use upon creation of the cluster.

Reviewer: ```suggestion
Initial set of repository certificates to be configured in Argo CD upon creation of the cluster.
```
